pinot-core/src/main/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPruner.java

```diff
@@ -262,6 +263,14 @@ private boolean pruneInPredicate(IndexSegment segment, InPredicate inPredicate,
         return false;
       }
     }
+
+    //Check for Bloom Filter
+    if(bloomFilter != null) {
```

Comment:
Please follow the code style: https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup

Reply:
updated!

Reply:
Can you please try auto-format? There should be space between if statement

Reply:
my bad, the auto format has some issues for me, will update!